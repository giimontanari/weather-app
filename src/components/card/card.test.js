import React from "react"
import renderer from 'react-test-renderer';
import Card from "./card"

describe("Check card structure with object required", () => {
    const weather = {
        dt: 31231231213,
        temp: {
            day: 213,
            min: 322,
            max: 432
        },
        humidity: 19
    }

    it('renders correctly', () => {
      const tree = renderer
        .create(<Card weather={weather} />)
          .toJSON();
        
        expect(tree).toMatchSnapshot();
    });
})

