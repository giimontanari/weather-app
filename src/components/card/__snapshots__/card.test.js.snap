// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check card structure with object required renders correctly 1`] = `
<div
  className="MuiPaper-root MuiCard-root sc-bdfBwQ hjsYJw MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="card-container"
  >
    <h1
      className="card-container-title"
    >
      miércoles 05 septiembre, 2959
    </h1>
    <h3
      className="card-container-temp"
    >
      -60
      °
    </h3>
  </div>
  <div
    className="card-others"
  >
    <span
      className="card-others-detail"
    >
      Temp. Min: 
      49
      °
    </span>
    <hr
      className="MuiDivider-root card-others-divider MuiDivider-middle MuiDivider-light"
    />
    <span
      className="card-others-detail"
    >
      Temp. Máx: 
      159
      °
    </span>
    <hr
      className="MuiDivider-root card-others-divider MuiDivider-middle MuiDivider-light"
    />
    <span
      className="card-others-detail"
    >
      Humedad: 
      19
      %
    </span>
  </div>
</div>
`;
